configure-mana

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh
 
 # These optional autoconf variables will be configured
-#     via contrib/mpi-proxy-split/Makefie_config.in
+#     via contrib/mpi-proxy-split/Makefile_config.in
+
+# Temporarily add /usr/sbin to the PATH in case ip can't be found.
+export PATH=$PATH:/usr/sbin
```

Comment:
We should be very careful on changing the global environment variables, if `ip` is the only one that has the problem why not just change that to `/usr/sbin/ip` ?

Another question, why `PATH` doesn't have `/usr/sbin` here? I think this should be there by default

Reply:
1. If `ip` is not in `/usr/sbin`, how your fix can fix the problem?
2. I don't know where you found that, it doesn't make sense to me is that adding `/usr/sbin` doesn't grant you any permission. How can that be a problem?

Reply:
1. If it's not in `PATH` or `/usr/sbin` then it means that the user doesn't have it and should install it.
2. I did a bit more digging and it depends on the OS; for instance Debian doesn't have it by default whereas CentOS does.

Reply:
Sorry that it took me so long to review this.

Here's what I suggest, @fyshhh:
1.  Delete the current changes that you have in configure-mana
2. In `configure-mana`, change, as below.
FROM:
```
MPI_ETHERNET_INTERFACE=\
`ip addr |grep -B1 link/ether | head -1 |sed -e 's%[^ ]*: \([^ ]*\): .*%\1%'`
```
TO:
```
MPI_ETHERNET_INTERFACE=\
`PATH=$PATH:/usr/sbin ip addr |grep -B1 link/ether | head -1 |sed -e 's%[^ ]*: \([^ ]*\): .*%\1%'`
```

This seems to accomplish your intended goal, while also not creating the danger mentioned by @jungan of adding `/usr/sbin` in the path of other commands.

When you've done this, please tag @jungan and me, so that we can give this a quick review.  Thanks.